tests/client-dbus/tests/udev/test_udev.py

```diff
@@ -133,7 +133,7 @@ def _wait_for_udev(expected_paths):
     expected. Always get the result of at least 1 Stratis enumeration.
     :param expected_paths: devnodes of paths that should belong to Stratis
     :type expected_paths: list of str
-    :return: None (May assert)
+    :return: None (May exit with a RuntimeError)
```

Comment:
Might as well fix this up to be proper header comment:

```
:return: None
:raises RuntimeError: 
```

Reply:
Resolved.